internal/generate_flags.go

```diff
@@ -0,0 +1,66 @@
+package internal
+
+import "github.com/urfave/cli/v2"
+
+// GenerateFlags returns the CLI flags for the generate command
+func GenerateFlags() []cli.Flag {
+	return []cli.Flag{
+		&cli.StringFlag{
+			Name:     "app-id",
+			Usage:    "GitHub App ID",
+			Required: true,
+			Aliases:  []string{"a"},
```

Comment:
What are your thoughts on maintaining the same parameters as the bash version? I like yours better and yet I'm thinking about the folks who will be migrating, and/or already have this hooked to some other stuff. This might be a breaking change for them.

![cs_2023-09-13_10-36-26](https://github.com/Link-/gh-token/assets/568794/287d3e0a-932b-4173-aca2-0d7a2df957fb)

Reply:
I have an idea that I think I can do one better. `urfave/cli` will accept multiple aliases. I'm good with keeping the same names as long as we replace the `_` with `-` since that is pretty much standard. So the default for App ID would be `--app-id` but it would have an alias of `--app_id` and `-i` to preserve backwards compatibility. And during the release announcement we can mention that all flags containing underscores have been preserved for backwards compatibility and are deprecated and may be removed in the future.

I'd also not be opposed to the `_` flags being the default, and the `-` flags being the alias and making the same announcement that `_` flags are deprecated and users should begin using `-` ASAP as the `_` flags may be removed in the future.

Thoughts on which route you'd rather go?

Reply:
Great, I like it. Let's do it :)

Reply:
I've added the old flags as aliases to preserve backwards compatibility, and updated the flags to match the names from `v1`